jekyll/_cci2/workflows.adoc

```diff
@@ -34,35 +34,50 @@ For example, if only one job in a workflow fails, you will know it is failing in
 Workflows may appear with one of the following states:
 
 [.table.table-striped]
-[cols=2*, options="header", stripes=even]
+[cols=3*, options="header", stripes=even]
 |===
-| State | Description
+| State | Description | Terminal State
 
 | RUNNING
 | Workflow is in progress
+| No
 
 | NOT RUN
 | Workflow was never started
+| Yes
 
 | CANCELED
 | Workflow was canceled before it finished
+| Yes
 
 | FAILING
-| A job in the workflow has failed
+| A job in the workflow has failed, but others are still running or yet to be approved
+| No
 
 | FAILED
 | One or more jobs in the workflow failed
+| Yes
 
 | SUCCESS
 | All jobs in the workflow completed successfully
+| Yes
 
-| NEEDS APPROVAL
+| NEEDS APPROVAL (UI) / ON HOLD
 | A job in the workflow is waiting for approval
+| No
 
-| NEEDS SETUP
+| ERROR
 | A workflow stanza is not included or is incorrect in the xref:configuration-reference#[`.circleci/config.yml` file] for this project
```

Comment:
It occurred to me that config issues are now caught higher up the stack. We'll get an error at the pipeline level if there are issues with the validity of the config. I had another look at this and the only time we set an `ERROR` status for workflows is when give up processing an `unblock-jobs` message after 10 minutes of retries, which in the majority cases is due to a failure building the permissions profile. It is quite rare.

```suggestion
| We experienced an internal error starting a job in the workflow
```